BunnymodXT/modules/HwDLL.hpp

```diff
@@ -199,6 +199,7 @@ class HwDLL : public IHookableNameFilterOrdered
 	}
 
 	HLStrafe::TraceResult PlayerTrace(const float start[3], const float end[3], HLStrafe::HullType hull, bool extendDistanceLimit = false);
+	HLStrafe::TraceResult CameraTrace(float max_distance = 1000.f);
```

Comment:
I think I usually used 8192 for this distance.
```suggestion
	HLStrafe::TraceResult CameraTrace(float max_distance = 8192);
```